containers/kubernetes/reference-content/managed-kubernetes-service-definition.mdx

```diff
@@ -0,0 +1,56 @@
+---
+meta:
+  title: Managed Kubernetes service definition
+  description: This page explains the definiton of the managed Kubernetes service at Scaleway
+content:
+  h1: Managed Kubernetes service definition
+  paragraph: This page explains the definiton of the managed Kubernetes service at Scaleway
+tags: kubernetes kubernetes-kapsule kapsule kosmos kubernetes-kosmos managed service definition
+dates:
+  validation: 2023-06-29
+  posted: 2023-06-29
+categories:
+  - kubernetes
+---
+
+Scaleway Kapsule and Kosmos are managed Kubernetes services. Scaleway's managed Kubernetes service abstracts away the complexities of managing and operating a Kubernetes cluster, enabling developers to focus on application development and deployment while ensuring a reliable and scalable environment for running containerized workloads.
+
+### What is a managed Kubernetes service?
+
+In the context of Scaleway, a managed Kubernetes service refers to the Kapsule and Kosmos products. Scaleway handles the management and maintenance of the Kubernetes control plane, along with all the crucial core components required for the proper operation of the Kubernetes cluster.
+
+### What components of Kubernetes are managed by Scaleway?
+
+Scaleway manages the Kubernetes control plane (either Kapsule or Kosmos), which consists of various components responsible for managing the cluster, scheduling applications, and maintaining cluster state.
+
+This includes components such as the control plane itself: etcd, API server, scheduler, cloud controller, and controller manager.
+
+Scaleway is also responsible for the operational upkeep of the operating system and system applications on the nodes. This includes essential components like CoreDNS, Kubeproxy, Container Networking Interface (CNI), and Container Storage Interface (CSI), which are vital for the optimal functioning of the Kubernetes cluster and its associated resources.
+
+### What responsibilities do I have as a user of a managed Kubernetes service?
+
+As a user, your responsibility lies primarily in deploying and managing the applications or services that you intend to run on the Kubernetes cluster.
+
+You are responsible for any deployments made to the cluster and the associated configurations.
+
+Furthermore, it is expected that you use the appropriate interfaces for managing resources, such as `kubectl` or the Scaleway Kubernetes console. However, please note that if you make any modification to the pre-installed components, such as Kubelet, at the node or system level as mentioned earlier, the responsibility for managing those components will be transferred to you.
```

Comment:
kubectl and the scaleway console/CLI are complementary not substitutable therefore I would use `and` instead of `or`

Reply:
```suggestion
Furthermore, it is expected that you use the appropriate interfaces for managing resources, such as `kubectl` and the Scaleway Kubernetes console. However, please note that if you make any modification to the pre-installed components, such as Kubelet, at the node or system level as mentioned earlier, the responsibility for managing those components will be transferred to you.
```